cinema/serializers.py

```diff
@@ -13,12 +83,22 @@
         return Movie.objects.create(**validated_data)
 
     def update(self, instance, validated_data):
-        instance.title = validated_data.get("title", instance.title)
+        instance.title = validated_data.get(
+            "title", instance.title
+        )
+
         instance.description = validated_data.get(
             "description", instance.description
         )
-        instance.duration = validated_data.get("duration", instance.duration)
+
+        instance.duration = validated_data.get(
+            "duration", instance.duration
+        )
 
         instance.save()
 
         return instance
+
+    class Meta:
+        model = Movie
+        fields = ("id", "title", "description", "duration")
```

Comment:
The `Meta` class is not applicable for `serializers.Serializer`. Consider using `serializers.ModelSerializer` if you want to specify the model and fields.